src/pip/_internal/cli/base_command.py

```diff
@@ -169,46 +170,60 @@ def _main(self, args: List[str]) -> int:
                 "This will become an error in pip 21.0."
             )
 
+        def intercepts_unhandled_exc(run_func):
+            # type: (Callable[..., int]) -> Callable[..., int]
```

Comment:
All types comments can be moved to annotations now. The tool `com2ann` can do this automatically if you'd like.

Reply:
Right, thanks. Old PR. Fixed!